### ISSUE TITLE
A vulnerability was found in the minimatch package in NX version 14.1.9

### ISSUE THREAD

**Issue by truongtphat** [nrwl/nx#15789]
### Current Behavior

NX version `14.1.9` is using `minimatch` version with vulnerability: `3.0.4`

### Expected Behavior

Update `minimatch` version to 3.0.5

### Steps to Reproduce

I'm using:
- NX version: `14.1.9`
- Angular version: `13.10.2`
- OWASP Dependency-Check to scan for vulnerable dependencies https://owasp.org/www-project-dependency-check/.

### Nx Report

```shell
Node : 16.10.0
   OS   : win32 x64
   yarn : 1.19.1

   nx : 14.1.9
   @nrwl/angular : 14.1.9
   @nrwl/cypress : 14.1.9
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.1.9
   @nrwl/eslint-plugin-nx : 14.1.9
   @nrwl/express : Not Found
   @nrwl/jest : 14.1.9
   @nrwl/js : Not Found
   @nrwl/linter : 14.1.9
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : Not Found
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.1.9
   @nrwl/web : Not Found
   @nrwl/workspace : 14.1.9
   typescript : 4.6.4
```

### Additional Information

I saw the merged PR https://github.com/nrwl/nx/pull/10472.
I tried to migrate @nrwl/workspace package from `14.1.9` to `14.2.1` but Angular's version will be affected.
Does anyone have any suggestions? Thank you so much.

**Comment by AgentEnder** [comment]
2 things are relevant here
- this is fixed in latest (as you noted)
- minimatch's vulnerability isn't relevant to Nx / nx users. The specific vulnerability is related to regex based ddos attacks, which are only relevant for servers. Nx is is a build tool, not a server environment, so you can safely ignore it.

**Comment by truongtphat** [comment]
Hi @AgentEnder,

Thanks for your response.

> * this is fixed in latest (as you noted)

I checked PR #10472 which was merged after [the released of NX 14.1.9](https://github.com/nrwl/nx/releases/tag/14.1.9). And the latest supported version for Angular 13.x is `14.1.9` ([Nx and Angular Version Compatibility Matrix](https://nx.dev/packages/angular/documents/angular-nx-version-matrix#nx-and-angular-version-compatibility-matrix)).

Do you have any plans to update `minimatch` to 3.0.5 to support Angular 13.x?

Thank you so much.
